.gitignore

```diff
@@ -0,0 +1,2 @@
+/tris-localserver
+/tris-testclient
```

Comment:
Trying to keep the root clean to help with upstreaming. Use `go install`, or `.git/info/exclude`.